src/databricks/labs/ucx/config.py

```diff
@@ -205,6 +205,8 @@ class WorkspaceConfig(_Config["WorkspaceConfig"]):
     connect: ConnectConfig | None = None
     num_threads: int | None = 10
     log_level: str | None = "INFO"
+    database_to_catalog_mapping: dict[str, str] = None
+    default_catalog: str = "ucx_default"
```

Comment:
```suggestion
    default_catalog: str = "main"
```

Reply:
Should be ucx_workspace_id more realistically, otherwise it will f-up customer data model if we run this on multiple workspace

Reply:
@FastLee @zpappa what should be the "default catalog" we're migrating into? we can't ask for 1:1 mapping between db and catalog.

Reply:
So, I'd like to provide a direct answer, but I'm failing to understand where this is in the process. 

Is this for repeated migrations scripts? I.e. I've already upgraded my table and now I need to keep syncing it?
Is this for initial migration? i.e. I've never upgraded my table and I need to sync it the first time.

Reply:
I think the process should look like:

**Initial assessment**
We scan all tables, generate in inventory, get all perms, etc. 
We provide a mapping file the customer can customize, but seed the UC table name with something like the workspace name or ID as the catalog.
Customer can choose to modify the mapping or not

**Initial migration**
We use the mapping file to sync tables, and persist the mapping data in inventory

**Repeated migration**
We use the mapping data in inventory to re-sync any tables

Reply:
Tracking this as https://github.com/databrickslabs/ucx/issues/320